### PR TITLE
Add jQuery.Flipster plugin w/ git auto-update with v1.1.2 assets

### LINKED ISSUE #12024: [Request] Add jQuery Flipster
**Library name:** jQuery Flipster
**Library description:** 
**Git repository url:** https://github.com/drien/jquery-flipster
**npm package name or url** (if there is one): https://www.npmjs.com/package/jquery.flipster
**License (List them all if it's multiple):** MIT
**Official homepage:** http://brokensquare.com/Code/jquery-flipster/demo/
**Wanna say something? Leave message here:** 

=====================

Notes from cdnjs maintainer(please remove this section after you read it):
Please read the [README.md](https://github.com/cdnjs/cdnjs#cdnjs-library-repository) and [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md) document first.

We encourage you to add a library via sending pull request,
it'll be faster than just opening a request issue,
since there are tons of issues, please wait with patience,
and please don't forget to read the guidelines for contributing, thanks!!

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/50504997-request-add-jquery-flipster?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by him2him2**
close #12024 
cc @drien

Pull request for issue: #12024
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/drien/jquery-flipster
 * Official website (optional, not the repository):  http://brokensquare.com/Code/jquery-flipster/demo/
 * NPM package url (optional): https://www.npmjs.com/package/jquery.flipster
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 40
   - Count of stars: 510
   - Count of forks: 195
 * NPM download stats (optional):
   - Downloads in the last day: 41
   - Downloads in the last week: 145
   - Downloads in the last month: 463

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery.flipster/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/drien/jquery-flipster.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com:drien/jquery-flipster.git",
+    "fileMap": [
+      {
+        "basePath": "dest",
```
Comment (maruilian11) [inline ajax/libs/jquery.flipster/package.json]: Should be `dist`? Thanks~

File: ajax/libs/jquery.flipster/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "jquery.flipster",
+  "version": "1.1.2",
+  "homepage": "",
```
Comment (maruilian11) [inline ajax/libs/jquery.flipster/package.json]: Let's use http://brokensquare.com/Code/jquery-flipster/demo/ as homepage. Thanks.

File: ajax/libs/jquery.flipster/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "ecosystem:jquery"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/drien/jquery-flipster.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com:drien/jquery-flipster.git",
```
Comment (maruilian11) [inline ajax/libs/jquery.flipster/package.json]: Please use `git://github.com/drien/jquery-flipster.git`, be care for ':'. Thanks!

Comment (him2him2) [reply ajax/libs/jquery.flipster/package.json]: @maruilian11 
I made the requested changes. Thank you!

**Pull request closed without merging**